google/cloud/storage/benchmarks/storage_throughput_vs_cpu_benchmark.cc

```diff
@@ -238,18 +237,20 @@ gcs_bm::ClientProvider MakeProvider(ThroughputOptions const& options) {
 #if GOOGLE_CLOUD_CPP_STORAGE_HAVE_GRPC
     using ::google::cloud::storage_experimental::DefaultGrpcClient;
     if (t == ExperimentTransport::kDirectPath) {
-      return DefaultGrpcClient(
-          opts.set<gcs_ex::GrpcPluginOption>("media")
-              .set<google::cloud::EndpointOption>(
-                  "google-c2p-experimental:///storage.googleapis.com"));
+      return DefaultGrpcClient(opts.set<gcs_ex::GrpcPluginOption>("media")
+                                   .set<google::cloud::EndpointOption>(
+                                       options.direct_path_endpoint));
     }
     if (t == ExperimentTransport::kGrpc) {
-      return DefaultGrpcClient(opts.set<gcs_ex::GrpcPluginOption>("none"));
+      return DefaultGrpcClient(
+          opts.set<gcs_ex::GrpcPluginOption>("media")
```

Comment:
Just pointing out that this changes from a `Client` to an `internal::HybridClient`

Reply:
Ack, thanks, intentional, and really fixing a bug.  I can separate this to another PR if you want